### PR TITLE
4-6 Release Notes for metering Report expiration

### PR DESCRIPTION
**Pull request by lbarbeevargas**
4.6 Release Notes for the new metering Report `expiration` field.

### CODE REVIEW

**Comment by lbarbeevargas** [comment]
Preview is available [here](http://file.bos.redhat.com/lbarbeev/09162020/4.6-release-notes-metering-report-expiration/release_notes/ocp-4-6-release-notes.html#ocp-4-6-metering-operator).

**Comment by lbarbeevargas** [comment]
@timflannagan1 @bentito PTAL for the 4.6 Release Notes. Thank you!

File: release_notes/ocp-4-6-release-notes.adoc [hunk 1]
```diff
@@ -196,6 +196,17 @@ By default, the following network connections are checked:
...
 ** the external API load balancer
 
+[id="ocp-4-6-operators"]
+=== Operators
+
+[id="ocp-4-6-metering-operator"]
+==== Support for retention period for metering Reports
+
+You can now set a retention period on a metering Report. The metering Report CustomResource
+has a new `expiration` field. If the `expiration` duration value is set on a Report,
```
Comment (pruan-rht) [inline release_notes/ocp-4-6-release-notes.adoc:206]: @lbarbeevargas @bentito perhaps an example here would benefit the end user.

Comment (bentito) [reply release_notes/ocp-4-6-release-notes.adoc:206]: I thought about that too but I think just release notes here, right?

File: release_notes/ocp-4-6-release-notes.adoc [hunk 2]
```diff
@@ -196,6 +196,17 @@ By default, the following network connections are checked:
 ** the internal API load balancer
 ** the external API load balancer
 
+[id="ocp-4-6-operators"]
+=== Operators
+
+[id="ocp-4-6-metering-operator"]
+==== Support for retention period for metering Reports
```
Comment (timflannagan1) [inline release_notes/ocp-4-6-release-notes.adoc:203]: I think there may be a missing word here as this reads a bit strange to me.

"Support for configuring a retention period of metering Reports" may be a bit wordy.

File: release_notes/ocp-4-6-release-notes.adoc (hunk 1 above)
Comment (lbarbeevargas) [reply release_notes/ocp-4-6-release-notes.adoc:206]: I can add a link to the docs if the user wants more information and an example.

File: release_notes/ocp-4-6-release-notes.adoc (hunk 2 above)
Comment (lbarbeevargas) [reply release_notes/ocp-4-6-release-notes.adoc:203]: Looking at the 4.5 release notes, we're not entirely consistent with how we write the titles for new support/features. Here's an example from 4.5 for Cluster logging - https://docs.openshift.com/container-platform/4.5/release_notes/ocp-4-5-release-notes.html#ocp-4-5-log-retention-policy

> **New Elasticsearch log retention feature**

What about, "Configuring a retention period of metering Reports" ?

I can get some guidance on wording during the peer review as well.

Comment (timflannagan1) [reply release_notes/ocp-4-6-release-notes.adoc:203]: Yeah, that's fine with me :+1:

**Comment by lbarbeevargas** [comment]
@openshift/team-documentation PTAL for 4.6 release notes. I commented out the link to the new Reports **expiration** section that will be available in 4.6.

File: release_notes/ocp-4-6-release-notes.adoc [hunk 3]
```diff
@@ -196,6 +196,19 @@ By default, the following network connections are checked:
 ** the internal API load balancer
 ** the external API load balancer
 
+[id="ocp-4-6-operators"]
+=== Operators
+
+[id="ocp-4-6-metering-operator"]
+==== Configuring a retention period of metering Reports
+
+You can now set a retention period on a metering Report. The metering Report CustomResource
```
Comment (adellape) [inline release_notes/ocp-4-6-release-notes.adoc:205]: s/CustomResource/custom resource/

File: release_notes/ocp-4-6-release-notes.adoc [hunk 4]
```diff
@@ -196,6 +196,19 @@ By default, the following network connections are checked:
...
+has a new `expiration` field. If the `expiration` duration value is set on a Report,
+and no other Reports or ReportQueries depend on the expiring Report, the Metering Operator
+removes the Report from your cluster at the end of its retention period. For more information, see metering 
+//../../metering/reports/metering-about-reports.adoc#metering-expiration_metering-about-reports[Reports expiration]
```
Comment (adellape) [inline release_notes/ocp-4-6-release-notes.adoc:209]: Looks like the `xref` got messed up here.

File: release_notes/ocp-4-6-release-notes.adoc [hunk 5]
```diff
@@ -196,6 +196,19 @@ By default, the following network connections are checked:
...
+.
```
Comment (adellape) [inline release_notes/ocp-4-6-release-notes.adoc:210]: Period on new line here, I think it's introducing a space.

**Review by adellape — COMMENTED** [review]
A few comments, mainly the `xref` one though.

File: release_notes/ocp-4-6-release-notes.adoc (hunk 4 above)
Comment (lbarbeevargas) [reply release_notes/ocp-4-6-release-notes.adoc:209]: @adellape, I commented out the xref because the link doesn't exist in the documentation/this feature branch yet. So the bad link would break the PR build.

File: release_notes/ocp-4-6-release-notes.adoc (hunk 5 above)
Comment (lbarbeevargas) [reply release_notes/ocp-4-6-release-notes.adoc:210]: This will be fixed when the link on the line above is uncommented. :)

**Comment by lbarbeevargas** [comment]
@adellape I commented out the last sentence with the xref and updated "CustomResource" to "custom resource." I can merge if it looks good to you. Thanks!

**Review by adellape — APPROVED** [review]
